examples/test-app/src/integration-test/groovy/testing/DomainMethodsTests.groovy

```diff
@@ -88,11 +90,11 @@ class DomainMethodsTests extends Specification {
             jump.name = 'fukt'
             jump.persist(flush: true)
             fail "it was supposed to fail the save because of validationException"
-        } catch (EntityValidationException e) {
+        } catch (EntityOptimisticLockingException e) {
             e.cause instanceof DataAccessException
             //assert e.cause instanceof org.springframework.orm.hibernate4.HibernateOptimisticLockingFailureException
             e.entity == jump
-            e.messageMap.code == 'default.not.saved.message'
+            e.messageMap.code == "Another user has updated the Jumper while you were editing"
```

Comment:
this can't be right, this can be the code like what it was ' default.not.saved.message' but should not be the message. I don't get it